src/applications/dependents/686c-674/config/chapters/stepchild-no-longer-part-of-household/removeChildHouseholdArrayPages.js

```diff
@@ -196,11 +196,29 @@ export const childAddressPage = {
     ...arrayBuilderItemSubsequentPageTitleUI(() => 'Stepchild’s address'),
     address: {
       ...addressUI({
+        title: '',
         labels: {
           militaryCheckbox:
             'This child lives on a United States military base outside of the U.S.',
         },
       }),
+      city: {
+        ...addressUI().city,
+        'ui:validations': [
+          (errors, city, formData) => {
+            const address = formData?.address;
```

Comment:
child address in full `formData`? Or is this formData only include the step child?